streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -425,6 +427,9 @@ public void flushInternal() {
     @Override
     public void close() {
         flush();
+        options.dispose();
+        wOptions.dispose();
+        fOptions.dispose();
```

Comment:
I noticed you nulled out `db`, but don't you need to do the same for the 3 `options` above?